proto/protocolapis/astria/protocol/genesis/v1alpha1/types.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+package astria.protocol.genesis.v1alpha1;
+
+import "astria/primitive/v1/types.proto";
+
+message GenesisAppState {
+  string chain_id = 1;
```

Comment:
This field is in preparation of a follow up refactoring sequencer to run `init-chain` with only the data provided by `GenesisAppState`.

Reply:
With this change we wouldn't additionally add a new `chainId` field into the genesis `app_state` would we?

Reply:
Yeah, we'd ultimately end up doing that. I had a dislike for extra configuration data being passed in "from the outside". I would like all genesis settings relevant to the app/sequencer to be in one place.

We would then either enforce that the cometbft chain ID match the sequencer chain ID - or ignore cometbft.

Reply:
The chain id is already part of the `InitChain` abci call, so this would be breaking expected abci conventions: https://github.com/cometbft/cometbft/blob/main/spec/abci/abci++_methods.md#initchain

This is inherently already a part of the call, I wouldn't want to break the abci convention here.

Reply:
Agree, in my mind these fields would just be duplicates: sequencer would check if its genesis contains the same chain ID that cometbft reports.